### DISCUSSION TITLE
nuxt 3 copy to clipboard

### DISCUSSION THREAD

**Discussion by borodadada** [nuxt/nuxt#16207] [Questions] [answered]
Hi! copy to clipboard it's possible ? 
I'm trying JS and it doesn't work.

```
<div id="copyline">{{myIp.ip}}</div>
```

```
    let copyText = document.getElementById("copyline").value;
    navigator.clipboard.writeText(copyText).then(() => {
        console.log('copyText: ', copyText)
    });
```

**Comment by wobsoriano** [comment]
https://vueuse.org/core/useClipboard/

https://vueuse.org/guide/#nuxt

**Comment by fathi91** [comment, marked as answer]
> Hi! copy to clipboard it's possible ? I'm trying JS and it doesn't work.
> 
> ```
> <div id="copyline">{{myIp.ip}}</div>
> ```
> 
> ```
>     let copyText = document.getElementById("copyline").value;
>     navigator.clipboard.writeText(copyText).then(() => {
>         console.log('copyText: ', copyText)
>     });
> ```

make sure that you are trying this on localhost because this feature is available only in secure contexts `HTTPS` it wont work if you are running `HTTP` server

**Reply by borodadada** [reply]
thx man all fine
